velox/serializers/PrestoSerializer.h

```diff
@@ -29,18 +29,26 @@ class PrestoVectorSerde : public VectorSerde {
 
     PrestoOptions(
         bool _useLosslessTimestamp,
-        common::CompressionKind _compressionKind)
+        common::CompressionKind _compressionKind,
+        bool nullsFirst = false)
```

Comment:
@oerling There is a comment from @mbasmanova internally that asks you to split the treatment of this new format to a separate diff